pkg/container/docker_run.go

```diff
@@ -579,6 +579,10 @@ func (cr *containerReference) copyDir(dstPath string, srcPath string, useGitIgno
 			// open files for taring
 			f, err := os.Open(file)
 			if err != nil {
+				if fi.Mode()&os.ModeSymlink == os.ModeSymlink {
+					// symlinks don't need to be copied, ignore this error
+					err = nil
+				}
```

Comment:
I agree with
> symlinks don't need to be copied

but act shouldn't even try to do that, it's just wrong to copy the content of a symlink into the tar. Act just had luck that docker ignored the content of the symlink.

I prefer to return just before line 579 on symlinks like
```golang
if fi.Mode()&os.ModeSymlink == os.ModeSymlink {
   return nil
}
```
instead of patching the error code, if it is a dangling symlink.